cucumber/api/features/authenticate.feature

```diff
@@ -114,11 +114,6 @@ Feature: Exchange a role's API key for a signed authentication token
       cucumber:user:alice successfully authenticated with authenticator authn
     """
 
-  Scenario: Authenticating with no Content-Type header succeeds without writing API key to the logs
-    And I save my place in the log file
-    And I can authenticate Alice with no Content-Type header
-    Then Alice's API key does not appear in the log
-
```

Comment:
Looks like very important test!

Reply:
Agreed. If this test is failing as part of the upgrade, let's find and fix the problem.

Reply:
@jvanderhoof @andytinkham 
Do you remember is there a special risk of API Key in logs appearance in case of "no Content-Type header".
@semyon-estrin Are there other tests checks API Key in logs "with Content Type header"

May be it worth to modify the test to some specific Content Type?

Reply:
Interesting!
This change may be breaking for some our customers or even for some our integrations.
Need to find a way to check it somehow.
CC: @eladkug

Reply:
@sashaCher please set as resolved/comment if needed.

Reply:
Resolved by @sashaCher